src/shared/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "shared"
+version = "0.0.1"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+candid = "0.9.3"
+ic-cdk = "0.10.0"
+ic-cdk-macros = "0.7.1"
+ic-cdk-timers = "0.4"
```

Comment:
I am surprised that timers are needed for the metrics.

Indeed, I think you can drop them.  I may have imported them wrongly in a previous PR.  Here are the unused dependencies, according to `udeps`:
```
$ cargo +nightly udeps
...
unused dependencies:
`airdrop v0.0.1 (/home/max/dfn/oisy-wallet/src/airdrop)`
└─── dependencies
     ├─── "ic-cdk-timers"
     └─── "ic-metrics-encoder"
`backend v0.0.1 (/home/max/dfn/oisy-wallet/src/backend)`
└─── dependencies
     └─── "ic-cdk-timers"
`shared v0.0.1 (/home/max/dfn/oisy-wallet/src/shared)`
└─── dependencies
     └─── "ic-cdk-timers"
```

Reply:
You can drop them in a separate PR though, if you wish.

Reply:
Copy / paste, you are right, absolutely no needed. Let me remove that

Reply:
Done